packages/kokoas-server/src/handleRequest/reqDownloadOrderToSupplier/createOrderDocument.ts

```diff
@@ -407,85 +163,140 @@ export const createOrderDocument = async (
       },
     );
 
-    // 単位
     drawText(
-      firstPage,
-      orderDetails[i].unit,
+      tgtPage,
+      vendorAddress2,
       {
-        x: 375,
-        y: posY,
+        x: 80,
+        y: 497,
         font: msChinoFont,
         size: 9,
       },
       {
         weight: 0.1,
-        align: 'center',
       },
     );
 
-    // 数量
-    const quentity = orderDetails[i].quantity.toFixed(2);
-
+    // 業者担当者1
     drawText(
-      firstPage,
-      quentity,
+      tgtPage,
+      vendorManager1,
       {
-        x: 388,
-        y: posY,
+        x: 80,
+        y: 478,
         font: msChinoFont,
         size: 9,
       },
       {
         weight: 0.1,
-        align: 'right',
       },
     );
 
-    // 単価
+    // 業者担当者2
     drawText(
-      firstPage,
-      orderDetails[i].costPrice.toLocaleString(),
+      tgtPage,
+      vendorManager2,
       {
-        x: 458,
-        y: posY,
+        x: 80,
+        y: 465,
         font: msChinoFont,
         size: 9,
       },
       {
         weight: 0.1,
-        align: 'right',
       },
     );
 
-    // 発注金額
+
+    // 工事名([工事番号]　工事名)
     drawText(
-      firstPage,
-      orderDetails[i].orderAmountBeforeTax.toLocaleString(),
+      tgtPage,
+      `[${projNumJa}] ${projName}`,
       {
-        x: 552,
-        y: posY,
+        x: 100,
+        y: 392,
         font: msChinoFont,
-        size: 9,
+        size: 10,
+      },
+      {
+        weight: 0.1,
+      },
+    );
+
+    // 工事場所住所
+    drawText(
+      tgtPage,
+      constAddress,
+      {
+        x: 100,
+        y: 379,
+        font: msChinoFont,
+        size: 10,
+      },
+      {
+        weight: 0.1,
+      },
+    );
+
+    // 工事期間
+    const constPeriod = getConstPeriod({
+      startDate: constStartDate,
+      finishDate: constFinishDate,
+    });
+    drawText(
+      tgtPage,
+      `${constPeriod}`,
+      {
+        x: 100,
+        y: 367,
+        font: msChinoFont,
+        size: 10,
       },
       {
         weight: 0.1,
-        align: 'right',
       },
     );
 
-    
-    // 税区分
-    const taxRate = orderDetails[i].taxRate === 0 ? '非課税' 
-      : `課税(${Big(orderDetails[i].taxRate).mul(100)
-        .toNumber()
-        .toString()}%)`;
 
+    // 会社情報
+    // 会社名
+    const companyNameArray = companyName.split(' ');
```

Comment:
変更不要。

以下の書き方も可能です。これで変数名で何を指しているか分かります。

```suggestion
    const [companyInfo, companyName]= companyName.split(' ');
```

Reply:
ご指摘ありがとうございます！
#1373 にて対応します。